salmon.md

```diff
@@ -0,0 +1,353 @@
+- asserts that `job_context` was created
+- asserts that the `job.success` is false
+
+### `test_salmon_quant_one_sample_double_reads`
+
+Tests salmon quant on a single sample with two original files (double reads)
+
+test steps:
+- calls `prepare_organism_indices`
+    - creates a dummy OrganismIndex that can be used in the salmon pipeline
+- creates the test sample and experiment
+- adds an additional fake sample to the experiment to prevent tximport from running on the experiment
+    - tximport will only run after salmon quant is run on every sample associated with the experiment
+- creates two original files and adds them to the test sample
+- calls `salmon._prepare_files` to prepare the original files before running salmon quant
+- calls `check_salmon_quant` to run salmon quant and tximport
+    - asserts quant output file is made
+    - asserts quant output file strongly correlates to reference file
+    - the files are strongly correlated if both columns #3 and #4 (zero-indexed) of the two files have a spearman correlation >= 0.99.
+- asserts that the test experiment is not ready for tximport yet
+
+### `test_salmon_quant_two_samples_single_read`
+
+Tests salmon quant on two samples each with one original file (single read) that belong to the same experiment
+
+test steps:
+- calls `prepare_organism_indices`
+    - creates a dummy OrganismIndex that can be used in the salmon pipeline
+- creates the test experiment
+- creates sample1 and og_file_1, associates the two, and adds the sample to the experiemnt
+- creates sample2 and og_file_2, associates the two, and adds the sample to the experiemnt
+- calls `salmon._prepare_files()`, passing in  og_file_1
+- calls `check_salmon_quant` to run salmon quant and tximport
+    - asserts quant output file is made
+    - asserts quant output file strongly correlates to reference file
+    - the files are strongly correlated if both columns #3 and #4 (zero-indexed) of the two files have a spearman correlation >= 0.99.
+- asserts that the experiment is not ready for tximport yet - salmon qunat still needs to be run on sample2
+- asserts that there are no tximport outputs yet
+- calls `salmon._prepare_files()`, passing in  og_file_2
+- cleans up tximport data from previous runs if it exists
+- calls `check_salmon_quant` to run salmon quant and tximport
+    - asserts quant output file is made
+    - asserts quant output file strongly correlates to reference file
+    - the files are strongly correlated if both columns #3 and #4 (zero-indexed) of the two files have a spearman correlation >= 0.99.
+    - at this point tximport should be run
+- asserts that the rds file (tximport output file) has been made
+- runs test_tximport.R to verify the output file
+- assert that two output files were created
+- assert that the two outut files exist
+
+### `test_get_tximport_inputs`
+
+Tests that tximport only considers RNA-Seq samples from GEO
```

Comment:
I would expect this to be SRA not GEO? https://github.com/AlexsLemonade/refinebio-processor-documentation/pull/11/files#diff-76b3c308d473d4219815890b7c1da498a41bd8124fce60ff30938fc6dfe03e6eR21

Reply:
Yeah I think that's correct - this is incorrectly documented in the refinebio test file as well